brainstormforce/ultimate-addons-for-gutenberg
# fix: Conditionally loaded blocks editor static css inside the spectra editor.

### Description
- In the Spectra Editor as the page renders inside an iframe the blocks editor static CSS does not load there, so in this PR I have made some changes in the config file & added conditional loading compatibility for the spectra editor.

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Check if Editor CSS loads in Spectra Editor
 - Check if Editor CSS loads in Core Editor on all devices.

### Checklist:
